### PR TITLE
Fix thumbnails and return timestamps

### PR DESCRIPTION
**Pull request by kidzegeye**
## Overview

Fixed thumbnail issue and made it so events' start and end dates were Unix timestamps instead of datetimes

## Changes Made

* Fixed thumbnail issue
* Replaced datetimes with timestamps
* Disabled model population in reset_db.sh

## Test Coverage

Tested via the api output and postgres database data

### CODE REVIEW

File: src/event/serializers.py [hunk 1]
```diff
@@ -20,3 +20,9 @@ def create(self, validated_data):
     class Meta:
         model = Event
         fields = ["id", "eatery", "event_description", "start", "end", "menu"]
+        
+    def to_representation(self, instance):
+        ret = super().to_representation(instance)
+        ret['start'] =round(datetime.strptime(ret['start'],'%Y-%m-%dT%H:%M:%S%z').timestamp())
+        ret['end'] = round(datetime.strptime(ret['end'],'%Y-%m-%dT%H:%M:%S%z').timestamp())
```
Comment (mateow99) [inline src/event/serializers.py]: Why do we need this for values that are already stored as timestamps?

**Review by mateow99 — APPROVED** [review]
just left one comment, but if it doesn't require changes then lgtm

File: src/event/serializers.py (hunk 1 above)
Comment (kidzegeye) [reply src/event/serializers.py]: They're stored in Iso8601 datetime format, this converts them to timestamps in the serialized output

Comment (mateow99) [reply src/event/serializers.py]: dont we get them from the dining json as timestamps? why not store them that way

Comment (kidzegeye) [reply src/event/serializers.py]: We get them from dining json like 2023-04-05T03:22:00+00:00Z

**Review by SashaLoayza — APPROVED** [review]
lgtm

**Pull request merged**